pkg/cmd/kafka/acl/create/create.go

```diff
@@ -179,27 +195,27 @@ func getRequestParams(opts *aclutil.CrudOptions) *requestParams {
 	}
 }
 
-func validateAndSetOpts(opts *aclutil.CrudOptions) error {
+func validateAndSetOpts(opts *aclutil.CrudOptions) string {
 
 	// user and service account should not be provided together
 	if userID != "" && serviceAccount != "" {
-		return opts.Localizer.MustLocalizeError("kafka.acl.common.error.bothPrincipalsSelected")
+		return opts.Localizer.MustLocalize("kafka.acl.common.error.bothPrincipalsSelected")
 	}
```

Comment:
```shell
❯ rhoas kafka acl grant-access
❌ Invalid flag combination:
 → no operation specified: must provide at least one of "--producer" or "--consumer" flag
 → principal is missing, provide one of "--user", "--service-account" or "--all-accounts" flags
 → "--topic" or "--topic-prefix" flag is required for consumer and producer operations
 → "--group" or "--group-prefix" flag is required for consumer operation.
```

I feel that we should not show the last two messages here regarding topic/group until either producer or consumer is chosen.

Reply:
I think we should show them, as issue suggests we should restrict running into errors again?

Reply:
Sure thing it makes sense. The message does specify the circumstances they are needed so you are 100% right.